.github/workflows/jacktrip.yml

```diff
@@ -275,7 +276,9 @@ jobs:
         shell: bash
         run: |
           choco install jack --version=1.9.17 --no-progress
-          choco install openssl --no-progress
+          if [[ "${{ matrix.build-system }}" == "qmake" && -n "${{ matrix.static-qt-version }}" && -n "${{ matrix.vcpkg-triplet }}" ]]; then 
+            vcpkg install openssl --triplet="${{ matrix.vcpkg-triplet }}"
+          fi
```

Comment:
Is this only needed when building Qt (i.e. not needed when cached qt build is used)? If so, we could consider moving this to the "build qt" step, which only runs if qt cache is not available. 
It's not a request, rather a suggestion for consideration. What do you think?

Reply:
I am also feeling this just in development on features! I think this is the right move.